data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/DispatcherEnv.java

```diff
@@ -31,12 +32,17 @@ public class DispatcherEnv extends BaseEnv {
     public static final String EGRESSES_INITIAL_CAPACITY = "EGRESSES_INITIAL_CAPACITY";
     private final int egressesInitialCapacity;
 
+    public static final String CE_METADATA_EXTENSION_PREFIX = "CE_METADATA_EXTENSION_PREFIX";
```

Comment:
was there any feedback from the node.js library on why they fail - even the spec says `should` ? 

CC @Cali0707 

Asking here, instead of adding all this code for pluggable prefixes

Reply:
The node.js library was updated successfully. We are in a tough position to update some of our legacy services so were hoping this change could be considered regardless.

Reply:
Yeah, we fixed it in the SDK since as you said @matzew  this was a "should" requirement. But, I guess that doesn't fix it if people are using a non-fixed version of the SDK